packages/editor/src/browser/editor-preferences.ts

```diff
@@ -23,7 +23,23 @@ import {
     PreferenceSchema,
     PreferenceChangeEvent
 } from '@theia/core/lib/browser/preferences';
-import { isOSX } from '@theia/core/lib/common/os';
+import { isWindows, isOSX } from '@theia/core/lib/common/os';
+
+const DEFAULT_WINDOWS_FONT_FAMILY = 'Consolas, \'Courier New\', monospace';
+const DEFAULT_MAC_FONT_FAMILY = 'Menlo, Monaco, \'Courier New\', monospace';
```

Comment:
I would ❤️ to see `'Fira Code'` leading that list!

Reply:
I think that should be explicitly enabled through user settings.